crates/ra_assists/src/assist_ctx.rs

```diff
@@ -84,6 +84,12 @@ impl<'a, DB: HirDatabase> AssistCtx<'a, DB> {
         f: impl FnOnce(&mut AssistBuilder),
     ) -> Option<Assist> {
         let label = AssistLabel { label: label.into(), id };
+        assert_eq!(
+            label.label.chars().nth(0).and_then(|c| Some(c.is_uppercase())).unwrap(),
```

Comment:
```suggestion
         assert!(label.label.chars().nth(0).unwrap().is_uppercase());
```

Reply:
Oh that's much better. Will do.

Reply:
It can also be `label.label.starts_with(char::is_uppercase)`, though this ignores empty strings.